packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -66,16 +65,18 @@ export class Relayer {
     readonly l2DeployData: { [key: string]: { blockNumber: number } },
     readonly l2LookbackWindow: number,
     readonly multicallBundler: MulticallBundler
-  ) {}
+  ) {
+    this.chainName = PublicNetworks[this.l2Client.chainId]?.name;
+  }
 
   async checkForPendingDepositsAndRelay(): Promise<void> {
-    this.logger.debug({ at: "AcrossRelayer#Relayer", message: "Checking for pending deposits and relaying" });
+    this.logger.debug({ at: `Relayer#${this.chainName}`, message: "Checking for pending deposits and relaying" });
```

Comment:
I think this is 🆗 , but as a minor note, it seems _slightly_ inconsistent with the fact that our relayer logs normally identify deposits by _chain id_ rather than chain name. I'm assuming you wanted to do the name here because it's the title of the log?

Reply:
I think chain ID is equally useful and in the sake of consistency we can use it over name